src/main/java/org/openstreetmap/atlas/geography/atlas/BareAtlas.java

```diff
@@ -484,6 +487,32 @@ public void saveAsGeoJson(final WritableResource resource, final Predicate<Atlas
         writer.close();
     }
 
+    @Override
+    public void saveAsLineDelimitedGeoJsonFeatures(final WritableResource resource,
+            final BiConsumer<AtlasEntity, JsonObject> jsonMutator)
+    {
+        saveAsLineDelimitedGeoJsonFeatures(resource, item -> true, jsonMutator);
+    }
+
+    @Override
+    public void saveAsLineDelimitedGeoJsonFeatures(final WritableResource resource,
+            final Predicate<AtlasEntity> matcher,
+            final BiConsumer<AtlasEntity, JsonObject> jsonMutator)
+    {
+        final JsonWriter writer = new JsonWriter(resource);
```

Comment:
How about putting this into a try-with-resources block?

Reply:
good idea, done!